R/LongitudinalGSF.R

```diff
@@ -87,17 +73,17 @@ LongitudinalGSF <- function(
 
         Parameter(name = "lm_gsf_a_phi", prior = a_phi, size = "n_arms"),
         Parameter(name = "lm_gsf_b_phi", prior = b_phi, size = "n_arms"),
-        Parameter(name = "lm_gsf_psi_phi", prior = psi_phi, size = "Nind"),
+        Parameter(name = "lm_gsf_psi_phi", prior = prior_none(), size = "Nind"),
```

Comment:
how would the user change the prior on this parameter now that we don't have it in the arguments anymore?